### PR TITLE
Add scripts and CI to help with the signing of windows releases

### PR DESCRIPTION
**Pull request by vxgmichel**
TODO:

- [x] Add readme

### CODE REVIEW

File: client/electron/package-sign.cmd
Comment (FirelightFlagboy) [inline client/electron/package-sign.cmd:1]: Put this file in `scripts`

Comment (FirelightFlagboy) [inline client/electron/package-sign.cmd:1]: I would name this file `sign-windows-package.cmd` or something like that :shrug:

File: client/electron/scripts/package-sign.ps1 [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+$node_version = node --version
+if (-Not ($node_version -eq $expected_node_version)) {
+    fnm use $expected_node_version
+}
+
+# Cleanup dist directory
+rm dist -r -force
+
+# Install node-modules
+npm install
```
Comment (FirelightFlagboy) [inline client/electron/scripts/package-sign.ps1:20]:
```suggestion
npm clean-install
```

**Review by FirelightFlagboy — CHANGES_REQUESTED** [review]

File: client/electron/scripts/package-sign.ps1 (hunk 1 above)
Comment (vxgmichel) [reply client/electron/scripts/package-sign.ps1]: Done!

File: client/electron/package-sign.cmd
Comment (vxgmichel) [reply client/electron/package-sign.cmd:1]: It has to run from the `client/electron` directory, so having the script there makes it more clear IMO.
What do you think?

**Review by Landeers — APPROVED** [review]

**Review by FirelightFlagboy — APPROVED** [review]

**Pull request merged**